src/dbcp/metadata/private_data_mart.py

```diff
@@ -0,0 +1,50 @@
+"""SQL Alchemy metadata for the private data mart tables."""
+
+from sqlalchemy import Boolean, Column, DateTime, Float, MetaData, String, Table
+
+metadata = MetaData()
+schema = "private_data_mart"
+
+fyi_projects_long_format = Table(
```

Comment:
Just for my understanding, the `fyi_projects_long_format` table feeds into `fyi_counties_proposed_clean_projects`, and I'm assuming you're persisting it in the new `private_data_mart` just for reference?